app/src/main/res/layout/item_search_result.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think this file should be called `item_search_speaker` as this is not even presented as a result necessarily (as per designs, before the user starts searching in this screen, we present them all the tracks, and speakers -- see https://zpl.io/Z2heIOl)

Also, in general, don't forget to extract styles (or leave TODOs for that)